### ISSUE TITLE
table-groups false positive

### ISSUE THREAD

**Issue by boris-petrov** [rwjblue/ember-template-lint#301]
```hbs
<table class='table table-hover table-striped'>
  <thead>
    <tr>
      <th></th>
    </tr>
  </thead>
  {{#some-component tagName='tbody' as |foo|}}
  {{/some-component}}
</table>
```

This gives a `Tables must have a table group (thead, tbody or tfoot).  table-groups` error.

**Comment by t-sauer** [comment]
Yes, this is expected. I think specifying the `tagName` in the template could be covered, but it is not possible to cover all cases (e.g. `tagName` is specified in the JS). 
The docs for the rule currently advise to disable the rule inline when you are sure, you are doing the right thing: https://github.com/rwjblue/ember-template-lint/blob/master/docs/rule/table-groups.md

**Comment by boris-petrov** [comment]
Ah, sorry, of course I didn't read the documentation. :) Just a question - is this something that is going to be fixed?

**Comment by rwjblue** [comment]
Unclear. I’m happy to accept a PR that addresses the specific situation (where a valid `tagName=“some valid tag”` is passed directly on invocation), but the general issue of a normal component invocation is not really possible (because we can’t know that it has the right tagName)...
